fsroot/lua/init.lua

```diff
@@ -140,8 +140,12 @@ local function fetch_searcher(extension, is_native)
 			local text = fetch_text(test_path)
 			if text then
 				if is_native then
-					error("Cannot load native modules from the web yet")
-					-- return package.loadlib(test_path, "luaopen_"..module:gsub("%.", "_"))
+					local lib, err = package.loadlib(test_path, "luaopen_"..module:gsub("%.so$", ""):gsub("%.", "_"))
+					if lib then
+						return lib
+					else
+						return "Failed to load lib "..test_path..": "..(err or "Unknown error")
```

Comment:
This should call `error`, not return a string

Reply:
I intentionally return a string just in case a user-defined loader that comes after can properly load it